app-modules/student-data-model/src/Filament/Resources/EducatableResource/Pages/Concerns/CanManageEducatableCareTeam.php

```diff
@@ -100,7 +100,7 @@ public function table(Table $table): Table
                             ->label('User')
                             ->searchable()
                             ->required()
-                            ->options(User::query()->tap(new HasLicense(Student::getLicenseType()))->pluck('name', 'id')),
+                            ->options(User::query()->tap(new HasLicense(Student::getLicenseType()))->whereDoesntHave('studentCareTeams')->pluck('name', 'id')),
```

Comment:
Users can be a part of multiple care teams. So this should be changed to specifically filter out Users that are already in the Educatables specific care team.